maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/SessionInfoHandler.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import FlyingFox
+import os
+import XCTest
+import Network
+
+@MainActor
+struct SessionInfoHandler: HTTPHandler {
+
```

Comment:
I see changes in xctest but the updated binaries are not here?

Reply:
How can I do that?

Reply:
(cd maestro; ./maestro-ios-xctest-runner/build-maestro-ios-runner.sh)

Reply:
@berikv this command fails for me with long output. I'll send to you over Slack

Reply:
You need to set up Xcode and install the 15.4 simulator runtime to use this command. Let me know if that resolves the error.